recipe/meta.yaml

```diff
@@ -1,40 +1,48 @@
+{% set org = "openforcefield" %}
 {% set name = "openff-qcsubmit" %}
-{% set version = "0.54.0" %}
+{% set version = "0.55.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/openforcefield/openff-qcsubmit/archive/{{ version }}.tar.gz
-  sha256: 858fff536df133019e0fe1a84f2f072e24f50108f34ca4730d767d0d9d7c8229
-
+  sha256: 016e387bbbb9619d528fe5e0a592a52dfb30ec20896c50ba13c02b28b5cce4f9
```

Comment:
FWIW both `url` and `sha256` are missing from our other feedstocks which do build fine: https://github.com/conda-forge/openff-interchange-feedstock/blob/b26acf10b6e0359245fbd8c88259382002c6b80f/recipe/meta.yaml#L9-L12